Doppler.Sap.Test/BillingServiceTest.cs

```diff
@@ -819,5 +839,89 @@ public async Task BillingService_Us_ShouldBeAddInvoiceWithoutPeriodicityProperty
                         .FreeText == "$ 15 -  Monthly Plan  - Period 00 0")),
                 Times.Once);
         }
+
+        [Fact]
+        public async Task BillingService_ShouldBeReturnInvocie_WhenTheRequestIsValid()
```

Comment:
Typo ShouldBeReturnInvocie should be ShouldBeReturnInvoice

Reply:
Done